app/views/trips/_sub_header.html.erb

```diff
@@ -1,13 +1,16 @@
 <nav class="sub-header">
   <ul>
-    <li id="current_trips_button" onclick="showCurrentTrips()">
-      <%= t("trips.index.current_trips_title") %>
+    <li>
+      <%= link_to t("trips.index.buttons.current"), current_trips_path,
```

Comment:
If this partial is only used in one view, a suggestion to just include the markup in that view. No need for another partial in that case

Reply:
Done in sha: `ddf36e5`